### PR TITLE
Issue1

### PR DESCRIPTION
**Pull request by pascalfoerster**
fixes issue #1 

We solved this by adding each instance of the cardinality into the tree.

Notes:
Constrains are not working -> new issue #35 
The configuration could be improved

### CODE REVIEW

**Comment by SundermannC** [comment]
The different attribute values are not shown when displaying configuration values in the feature-model editor.

image

This seems fine for this first version but should be addressed sometime in the future.

**Comment by SundermannC** [comment]
The .json configuration format seems broken:

image

I manually changed the values in the configuration process and the values are still empty. Also, the linter shows indicates errors in the config.json

File: uvls/src/core/module.rs [hunk 1]
```diff
@@ -179,43 +180,102 @@ impl Module {
...
+                                                    path.range(),
+                                                    format!(
+                                                        "expected {} got {}",
+                                                        file.type_of(sym).unwrap(),
+                                                        val.ty()
+                                                    ),
+                                                );
+                                            }
+                                        } 
+                                        Symbol::Attribute(i) => {
```
Comment (TobiUUlm) [inline uvls/src/core/module.rs:246]: this value `i` is unused, i get a warning. please change it to `_` or `..`.

**Review by TobiUUlm — COMMENTED** [review]
I can not test this properly, which is probably due to my Windows machine. I think the merge of 
#38 could help here.  
But the configuration works :)

**Comment by SundermannC** [comment]
The configuration editor and the output configurations look good. However, I think the reasoning for cardinality in combination with tree hierarchy is not optimal.

image

results in

image

here I would expect that not picking any Brake would be valid since it is optional. 

Similarly, if the brake is changed to mandatory every Brake instance needs to be selected. Here, I would expect that between 1 and 3 need to be selected and not necessarily all.

**Comment by felixrieg** [comment]
I totally agree with that, but could we make this a new feature?

**Comment by SundermannC** [comment]
Ok, then I will merge this PR for now. @felixrieg, can you create the issue?

**Pull request merged**